packages/react-atlas-default-theme/src/table/index.js

```diff
@@ -1,2 +1,3 @@
+import "./reactBsTable.global.css";
```

Comment:
I do have a CSS call table.css in the same folder, it's for local styling css loader. Using css module to pass moduled class into bs-table (by attributes of BootstrapTable components like containerClass/tableContainerClass)

The global css is for styling some classes hard code in bs-table.